service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/CaseSubmissionController.java

```diff
@@ -76,9 +78,20 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToStartEvent(
 
         return AboutToStartOrSubmitCallbackResponse.builder()
             .data(data)
+            .errors(validate(mapper.convertValue(data, CaseData.class)))
             .build();
     }
 
+    private List<String> validate(CaseData caseData) {
```

Comment:
validateLocalAuthority?

Reply:
I made it generic in case more validation is needed in the future. I am not sure it is worth wait for another build.